packages/dashboard-message-bus-common/package.json

```diff
@@ -5,8 +5,11 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "files": [
-    "dist/lib"
+    "dist"
   ],
+  "directories": {
+    "dist": "dist"
```

Comment:
I don't understand what this is doing... I can't even find documentation for `directories.dist` in the [package.json documentation](https://docs.npmjs.com/cli/v6/configuring-npm/package-json).  What is the intention of this?  Should this be here?

Reply:
Actually it looks like a bunch of our `package.json` files do weird things with `directories` that I don't understand.  (Including some of mine, due to my use of copy-paste...)  So uh it seems likely I just don't know what I'm talking about here. :P  Or alternatively maybe we should clean that up later...

Reply:
A clean-up after may be necessary. I don't understand the purpose of `directories`, but since `files` and `directories` are the two relevant `package.json` properties that are declared in some other Truffle packages, which are installable and importable, I thought it'd be prudent to include both.

I'm pretty sure `directories` can be omitted though 😜